### ISSUE TITLE
ExLoss cannot  converge

### ISSUE THREAD

**Issue by whut2962575697** [Yu-Wu/One-Example-Person-ReID#7]
Thank you for your great work!But i found the exloss cannot converge when training, the optimizer is adam, lr=3e-4

**Comment by Yu-Wu** [comment]
@whut2962575697 

Thanks for your interest in our paper. 

I have never tried the Adam optimizer. I only tried SGD optimizer in our experiments. Maybe the learning rate should be tuned for Adam, since Adam is very sensitive to lr.
